templates/page.html.twig

```diff
@@ -1,26 +1,27 @@
-{%- include "@saa_victoria_subtheme/components/brand-bar/saa--brand-bar.html.twig" with { modifier_class : "su-brand-bar--dark" } -%}
+{#
+/**
+ * Page.html.twig from stanford_basic.
+ */
+#}
+{%- extends "@stanford_basic/page.html.twig"  -%}
```

Comment:
@katrialesser you don't need to include `templates` in the path of the namespace. That is assumed in most cases.

Reply:
Gotcha, thank you!